libmscore/rendermidi.cpp

```diff
@@ -233,6 +243,45 @@ static void playNote(EventMap* events, const Note* note, int channel, int pitch,
       ev.setNote(note);
       if (offTime < onTime)
             offTime = onTime;
+      // adds portamento for continuous glissando
+      for (Spanner* spanner : note->spannerFor()) {
+            if (spanner->type() == ElementType::GLISSANDO) {
+                  Glissando *glissando = toGlissando(spanner);
+                  GlissandoStyle glissandoStyle = glissando->glissandoStyle();
+                  if (glissandoStyle == GlissandoStyle::PORTAMENTO) {
+                        //Changes the notes pitch to the next note's pitch, note-on is the eventual pitch
+                        Note* nextNote = toNote(spanner->endElement());
+                        ev.setPitch(nextNote->pitch());
+
+                        int time = toMilliseconds(note->score()->tempo(note->tick()), offTime - onTime);
+                        int lsb = (time & 0x00FF);
+                        time = (time >> 8);
+                        int msb = (time & 0x00FF);
+
+                        NPlayEvent portamentoOn(ME_CONTROLLER, channel, CTRL_PORTAMENTO, MIDI_ON_SIGNAL);
+                        portamentoOn.setOriginatingStaff(staffIdx);
+                        NPlayEvent portamentoControl(ME_CONTROLLER, channel, CTRL_PORTAMENTO_CONTROL, pitch);
+                        portamentoControl.setOriginatingStaff(staffIdx);
+                        NPlayEvent portamentoTimeMSB(ME_CONTROLLER, channel, CTRL_PORTAMENTO_TIME_MSB, msb);
+                        portamentoTimeMSB.setOriginatingStaff(staffIdx);
+                        NPlayEvent portamentoTimeLSB(ME_CONTROLLER, channel, CTRL_PORTAMENTO_TIME_LSB, lsb);
+                        portamentoTimeLSB.setOriginatingStaff(staffIdx);
+                        
+                        ev.setPortamento(true);
+                        if (onTime == 0)
+                              onTime++;
+                        events->insert(std::pair<int, NPlayEvent>(onTime-1, portamentoOn));
```

Comment:
See discussion at https://github.com/musescore/MuseScore/pull/5853#pullrequestreview-381727058

Reply:
and this